tests/integration/routes/test_session.py

```diff
@@ -41,7 +41,7 @@ def test_session_signed_out_no_cookie_session_default_config(self):
         self.deleteCookie()
         self.get("/sign-out", follow_redirects=False)
 
-        self.assertInRedirect("ons.gov.uk")
+        self.assertInRedirect("surveys.ons.gov.uk")
```

Comment:
Probably should use the value from settings

Reply:
Makes more sense, pushed. Thanks!